### PR TITLE
Add new fields to `fair create` yaml output

### LINKED ISSUE #25: fair create does not add some elements to config.yaml
`fair create` does not add these to the default `run_metadata` block:

- `remote_data_registry_url`
- `remote_repo`
- `script`, which should be echo "Hello World!" by default

These names are also wrong:

- `local_data_registry` should be `local_data_registry_url`
- `remote_data_registry` should be `remote_data_registry_url`

### PR DESCRIPTION
**Pull request by DennisReddyhoff**
Fixes #25

### CODE REVIEW

**Comment by DennisReddyhoff** [comment]
Fixes issue #25 @richardreeve

File: fair/templates/config.jinja [hunk 1]
```diff
@@ -1,7 +1,10 @@
 run_metadata:
   description: Project description
```
Comment (richardreeve) [inline fair/templates/config.jinja:2]: I don't know whether this is the right place to put this comment in, but given this is a dummy entry in the yaml file, I wonder if it makes sense to put this line at the bottom just before the `script` key. I might move the `default_input_namespace` and `default_output_namespace` down as well (but above `description`), as they could possibly be changed. The rest are probably never going to be edited.

**Review by richardreeve — COMMENTED** [review]
Happy for you to merge this, but maybe reordering these fields in the yaml file being created will make it a bit more obvious which ones people should edit?

**Comment by richardreeve** [comment]
BTW @DennisReddyhoff it seems like you have to put the `Fixes #25` statement in the first comment of the PR to get GitHub to recognise it should automatically close the associated issue. I've edited you comment just to check, and it works now...

**Pull request merged**